candi/control-plane-kubeadm/config.yaml.tpl

```diff
@@ -173,16 +177,32 @@ etcd:
   {{- end }}
 {{- end }}
 ---
+{{- if semverCompare ">= 1.22" .clusterConfiguration.kubernetesVersion }}
+apiVersion: kubeadm.k8s.io/v1beta3
+{{- else }}
 apiVersion: kubeadm.k8s.io/v1beta2
+{{- end }}
 kind: InitConfiguration
+{{- if semverCompare ">= 1.22" .clusterConfiguration.kubernetesVersion }}
+patches:
+  directory: /etc/kubernetes/deckhouse/kubeadm/patches/
```

Comment:
I think use of command-line key for patches is more clear way.

Reply:
No way anymore :))
`--patches` flag couldn't be used with `--config` flag. Only one way to apply patches now - add them into the config